setup.cfg

```diff
@@ -1,6 +1,7 @@
 [pycodestyle]
 max_line_length=120
 exclude=migrations
+ignore=W0613
```

Comment:
If you need to disable this, use `# pylint: disable=unused-argument` at the appropriate location. Do not ignore warnings globally without a discussion. Also, this won't work because Pylint is configured in `pylintrc`.